addons/web/static/src/core/commands/command_palette.xml

```diff
@@ -3,25 +3,25 @@
 
   <t t-name="web.CommandPalette" owl="1">
     <div>
-      <div class="o_command_palette_search">
-        <input type="text" t-model="state.searchValue" t-on-input="onSearchInput" autofocus="" t-att-placeholder="state.placeholder"/>
-        <i  t-att-title="state.placeholder" role="img"  t-att-aria-label="state.placeholder" class="fa fa-search"></i>
+      <div class="o_command_palette_search d-flex align-items-center justify-content-between bg-white px-4 py-3 mb-2">
+        <input class="border-0 p-0" type="text" t-model="state.searchValue" t-on-input="onSearchInput" autofocus="" t-att-placeholder="state.placeholder"/>
+        <i  t-att-title="state.placeholder" role="img"  t-att-aria-label="state.placeholder" class="fa fa-search text-black-25"></i>
       </div>
 
-      <div t-ref="listbox" class="o_command_palette_listbox">
-        <div t-if="!state.commands.length" class="o_command_palette_listbox_empty" t-esc="state.emptyMessage"/>
+      <div t-ref="listbox" class="o_command_palette_listbox position-relative overflow-auto">
+        <div t-if="!state.commands.length" class="o_command_palette_listbox_empty px-4 py-3 font-italic" t-esc="state.emptyMessage"/>
         <t t-if="!isFuzzySearch">
           <t t-foreach="commandsByCategory" t-as="category">
 
-            <div class="o_command_category" t-key="category.keyId">
+            <div class="o_command_category px-0" t-key="category.keyId">
               <t t-foreach="category.commands" t-as="command">
                 <t t-set="commandIndex" t-value="state.commands.indexOf(command)" />
-                <div t-attf-id="o_command_{{commandIndex}}" class="o_command" t-att-class="{ focused: state.selectedCommand === command }" t-on-click="onCommandClicked(commandIndex)" t-on-mouseenter="onCommandMouseEnter(commandIndex)" t-key="command.keyId">
-                  <t t-component="command.Component || DefaultCommandItem" name="command.name" searchValue="clearSearchValue" t-props="command.props" t-on-close="props.closeMe()" t-on-execute-command="executeCommand(command)">
+                <div t-attf-id="o_command_{{commandIndex}}" class="o_command position-relative" t-att-class="{ focused: state.selectedCommand === command }" t-on-click="onCommandClicked(commandIndex)" t-on-mouseenter="onCommandMouseEnter(commandIndex)" t-key="command.keyId">
```

Comment:
I'm wondering if we can use `list-group-flush` ([ref](https://getbootstrap.com/docs/4.3/components/list-group/#flush)) and remove even more...

Reply:
Haven't been able to do this.